### ISSUE TITLE
Add event callback to datagrid when a cell is navigated

### ISSUE THREAD

**Issue by mtbayley** [Megabit/Blazorise#5661]
Is it possible to create an event callback when the DataGrid is navigated via a click or keyboard (arrow keys)? It would return the row object as part of the event args. Bonus if it can also return name of the field in the row clicked.

There is the RowClicked event which captures the click, but does not handle the kayboard navigation.

**Comment by David-Moreira** [comment]
Everything's possible! haha

Would you be so kind to provide information behind that you're trying to do with listening to these events? Perhaps there could be other ways to solve that problem?

**Comment by stsrki** [comment]
I think this can be done as part of the https://github.com/Megabit/Blazorise/issues/5295

**Comment by mtbayley** [comment]
> Would you be so kind to provide information behind that you're trying to do with listening to these events? Perhaps there could be other ways to solve that problem?

I would like to display a PDF associated with the row. As the user navigates with cells, the PDF associated with the selected cell would display.

**Comment by David-Moreira** [comment]
> > Would you be so kind to provide information behind that you're trying to do with listening to these events? Perhaps there could be other ways to solve that problem?
> 
> I would like to display a PDF associated with the row. As the user navigates with cells, the PDF associated with the selected cell would display.

Fair enough, I think https://github.com/Megabit/Blazorise/issues/5295 - `SelectedCellChanged` is what you want. Anyway it probably won't hurt to introduce the event for further flexibility, we'll let you know once we tackle the feature.

**Comment by stsrki** [comment]
I think that FocusedCellChanged can also be introduced.
